kibana-reports/public/components/main/reports_table.tsx

```diff
@@ -37,6 +38,8 @@ import {
   humanReadableDate,
   generateReportById,
 } from './main_utils';
+import dateMath from '@elastic/datemath';
+import uuidv4 from 'uuid/v4';
```

Comment:
in the server side we use v1

Reply:
v1 is based on timestamp, v4 is random. I don't think it will make a big difference